Testreglar/1.2.1/Nett/nett-1.2.1b.json

```diff
@@ -0,0 +1,249 @@
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Video er direkte/live."
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Er videoen et mediealternativ til tekst, og tydelig merket som det?",
+			"ht": "<p>For å være mediealternativ skal videoen:</p><ul><li>ikke presentere mer informasjon enn det som gis via tekst og</li><li>være tydelig merket. Det vil si<ul><li>være plassert i direkte tilknytning til aktuell tekst og</li><li>gi informasjon som viser at det er et mediealternativ</li></ul></li></ul><p>Slik tester du:</p><ul><li>Undersøk om det i tilknytning til videoen, er en tekst som fremstår som hovedformatet for formidling av informasjonen</li><li>Momenter som tilsier at videoen er et mediealternativ til tekst kan være:<ul><li>Ikon, for eksempel kamera-ikon</li><li>Lenke med lenketekst som viser til en video, for eksempel \"Se opptak\", \"Animasjon\"</li></ul></li><li>Sjekk at videoen ikke presenterer mer informasjon enn det som gis via teksten</li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Video uten lyd er mediealternativ til tekst."
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				}
+			}
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "Er det en tekst visuelt plassert nær videoen/avspilleren som omtaler videoen?",
+			"ht": "<p>Slik tester du:</p><ul><li>Gjør en visuell inspeksjon</li><li>Sjekk om du finner et tekstalternativ nær videoen (på samme nettside), for eksempel rett over eller under videoen.</li></ul><p><strong>Merk:</strong> Du skal ikke ta stilling til kvaliteten på teksten i dette steget.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.7"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				}
+			}
+		},
+		{
+			"stegnr": "3.6",
+			"spm": "Ligger det en mekanisme i nærheten av videoen, som går til et tekstalternativ for videoen?",
+			"ht": "<p>Slik tester du:</p><ul><li>Gjør en visuell inspeksjon</li><li>Sjekk om det ligger mekanisme (lenke, knapp eller lignende) i nærheten av videoen som går til tekstalternativet<ul><li>Eksempel på plassering er rett over eller under videoen</li></ul></li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.7"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.9",
+					"delutfall": {
+						"nr": 0,
+						"tekst": "Video uten lyd har ikke tekstalternativ.",
+						"fasit": "Nei"
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Er tekstalternativet kodet som tekst?",
```

Comment:
Alternativ 3, inspiser elementet og sjekk at dette er kodet slik at det støtter text content og er skrevet som tekst

Reply:
De to alternativer fungerte fint. Tror ikke om vi trenger flere alternativer her.